jobs/build/sync-for-ci/Jenkinsfile

```diff
@@ -80,6 +80,8 @@ node {
     LOCAL_SYNC_DIR = "${REPOSYNC_BASE_DIR}/${REPOSYNC_DIR}"
     MIRROR_TARGET = "use-mirror-upload.ops.rhcloud.com"
     MIRROR_RELATIVE_REPOSYNC = "reposync/${REPOSYNC_DIR}"
+    def (MAJOR, MINOR) = [0, 0]
+    (MAJOR, MINOR) = commonlib.extractMajorMinorVersionNumbers(params.REPOSYNC_DIR)
```

Comment:
REPOSYNC_DIR isn't always a version number, so I think this will fail on e.g. `ci-deps` runs of this script. In that case it doesn't make sense to figure out the next anyway.